spec/support/pundit.rb

```diff
@@ -0,0 +1,17 @@
+module PunditViewPolicy
+  extend ActiveSupport::Concern
+
+  included do
+    before do
+      controller.singleton_class.class_eval do
+        def policy(instance)
+          Class.new do
+            def method_missing(*args, &block); true; end
```

Comment:
I dug around and around the web on this one. But yes, this makes all view specs show everything. Without this, the `policy(model)` method is not found, which is worse.